index.html

```diff
@@ -4,16 +4,39 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="menu.css">
```

Comment:
having many style sheets linked is ok for now... in the future, we will see how to combine those two for page optimization